packages/react/error-boundary/src/ErrorBoundaryGroup.tsx

```diff
@@ -1,18 +1,11 @@
 /** @tossdocs-ignore */
-import {
-  ComponentPropsWithoutRef,
-  ComponentType,
-  createContext,
-  ReactNode,
-  useContext,
-  useEffect,
-  useMemo,
-  useRef,
-} from 'react';
+import { ComponentType, createContext, ReactNode, useContext, useEffect, useMemo, useRef } from 'react';
 import { useIsMounted, useKey } from './hooks';
+import { ComponentPropsWithoutChildren } from './types';
```

Comment:
I added ComponentPropsWithoutChildren to repetitive implementation of ComponentProps without children to make type of HOC's 2nd arg